packages/flutter/lib/src/material/about.dart

```diff
@@ -384,6 +463,122 @@ class AboutDialog extends StatelessWidget {
+  });
+
+  List<Widget>? _actions(BuildContext context) {
+    final ThemeData themeData = Theme.of(context);
+    final MaterialLocalizations localizations =
+        MaterialLocalizations.of(context);
+    switch (themeData.platform) {
+      case TargetPlatform.iOS:
+      case TargetPlatform.macOS:
+        return <Widget>[
+          CupertinoDialogAction(
+            child: Text(themeData.useMaterial3
+                ? localizations.viewLicensesButtonLabel
+                : localizations.viewLicensesButtonLabel.toUpperCase()),
+            onPressed: () {
+              showLicensePage(
+                context: context,
+                applicationName: applicationName,
+                applicationVersion: applicationVersion,
+                applicationIcon: applicationIcon,
+                applicationLegalese: applicationLegalese,
+              );
+            },
+          ),
+          CupertinoDialogAction(
+            child: Text(themeData.useMaterial3
+                ? localizations.closeButtonLabel
+                : localizations.closeButtonLabel.toUpperCase()),
+            onPressed: () {
+              Navigator.pop(context);
+            },
+          ),
+        ];
+      case TargetPlatform.android:
+      case TargetPlatform.fuchsia:
+      case TargetPlatform.linux:
+      case TargetPlatform.windows:
+        return <Widget>[
+          TextButton(
+            child: Text(themeData.useMaterial3
+                ? localizations.viewLicensesButtonLabel
+                : localizations.viewLicensesButtonLabel.toUpperCase()),
+            onPressed: () {
+              showLicensePage(
+                context: context,
+                applicationName: applicationName,
+                applicationVersion: applicationVersion,
+                applicationIcon: applicationIcon,
+                applicationLegalese: applicationLegalese,
+              );
+            },
+          ),
+          TextButton(
+            child: Text(themeData.useMaterial3
+                ? localizations.closeButtonLabel
+                : localizations.closeButtonLabel.toUpperCase()),
+            onPressed: () {
+              Navigator.pop(context);
+            },
+          ),
+        ];
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    super.build(context);
+
+    final String name = applicationName ?? _defaultApplicationName(context);
+    final String version =
+        applicationVersion ?? _defaultApplicationVersion(context);
+    final Widget? icon = applicationIcon ?? _defaultApplicationIcon(context);
+    final ThemeData themeData = Theme.of(context);
+    final List<Widget>? actions = _actions(context);
+
+    return AlertDialog.adaptive(
+      content: ListBody(
+        children: <Widget>[
+          Row(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: <Widget>[
+              if (icon != null)
+                IconTheme(data: themeData.iconTheme, child: icon),
```

Comment:
Nit: maybe on the same line as the if? especially since there are no curly braces.